### ISSUE TITLE
css import in <style> tag 

### ISSUE THREAD

**Issue by monkoose** [babichjacob/sapper-postcss-template#17]
Hello. Thank you for this template.

I have some troubles making this template working with external css styles, as example [swiper.js](https://github.com/nolimits4web/swiper)

If i import css in a <script> tag with `import "swiper/swiper-bundle.css";` al works fine, but definitely it doesn't processes and outputs as it is.
When i put `@import "swiper/swiper-bundle.css` into `<style lang="postcss">` or `<style lang="postcss" global>` or `<style global>` or into global.pcss file and then import it as in template, then on a dev environment it is working well (except without global  there are a lot of warnings about unused css), but when i export with `npm run export` and then run `npx serve __sapper__/export` all library related css is gone from main.css. 

It's definitely because i miss something. Have you understood my problem, can you point me into the direction how to fix this?

Thank you one more time.

**Comment by babichjacob** [comment]
👋 Hi, thanks for the interest in the project. Before I research any further, here's a quick diagnosis and some possible solutions:

The problem is that PurgeCSS is not safelisting the import.

1. Try putting [`/* purgecss start ignore */` before the `@import` statement and `/* purgecss end ignore */`](https://purgecss.com/whitelisting.html#in-the-css-directly) after it.
2. There may be some way to modify [PurgeCSS options in `postcss.config.js`](https://github.com/babichjacob/sapper-postcss-template/blob/main/postcss.config.js#L24-L24) to either [`whitelist` or `whitelistPatterns`](https://purgecss.com/whitelisting.html) the classes used in that file, or update the `content` to include the exact path in `node_modules` to the file.

**Comment by babichjacob** [comment]
> If i import css in a <script> tag with `import "swiper/swiper-bundle.css";`

I recommend against this.

**Comment by monkoose** [comment]
> I recommend against this.

Yes, i know. Have read one of your comments in the issues of the sapper repo.

Ok i will trry your advices around purgecss. Thank you.

**Comment by monkoose** [comment]
TL;DR `whitelistPatterns` with `[/^swiper-/]` pattern solved this issue.

Another investigation.
Wrapping into `/* purgecss start ignore */` `/* purgecss end ignore */` doesn't help, even with `/*! ` as suggested on purgecss documentation.
